src/platform/tests/BUILD.gn

```diff
@@ -22,6 +22,7 @@ declare_args() {
   # Platform DNSsd should always be running on darwin. On Linux
   # this depends on avahi being available and running.
   chip_enable_dnssd_tests = chip_device_platform == "darwin"
+  chip_enable_dnssd_tests = chip_device_platform == "tizen"
```

Comment:
does this override the value above? Why is that ok?

Reply:
It's not.